gridstatus/eia.py

```diff
@@ -775,11 +775,14 @@ def _handle_generator_data(
                 )
 
         for col in GENERATOR_INT_COLUMNS:
-            if col in df.columns and df[col].dtype == "object":
```

Comment:
`Entity ID` for some of the datasets was returned as a float. The issue was this conditional did not account for that column being a float in the dataset before conversion. Entity ID should be an integer looking at the raw data.